raisinbread/instruments/EEG_Acquisition_form.linst

```diff
@@ -0,0 +1,97 @@
+testname{@}EEG_Acquisition_form
+table{@}EEG_Acquisition_form
```

Comment:
since those are in the .meta file they don't need to be in here

Reply:
Lines removed